### PR TITLE
[WIP] AutoML Tables

### PR DESCRIPTION
**Pull request by sirtorry**
todo: tests, readme, requirements, resources

### CODE REVIEW

**Comment by sirtorry** [comment]
"the Readme in github for the product stockout. It looks like the images links got changed and we have broken images. the license also does not show well vs other notebook readme"

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 1]
```diff
@@ -0,0 +1,1204 @@
...
+        "import tarfile\n",
+        "\n",
+        "# Upload the client library\n",
+        "compressed_file_upload = files.upload()\n",
+        "compressed_file_name = list(compressed_file_upload.keys())[0]\n",
+        "# Decompress the client library\n",
+        "with tarfile.open(compressed_file_name) as tar:\n",
+        "  tar.extractall(path='.')\n",
+        "# Install the client library\n",
+        "!pip install ./python"
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace this line with:
`!pip install google-cloud-automl`

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 2]
```diff
@@ -0,0 +1,1204 @@
...
+        "from google.colab import files\n",
+        "import tarfile\n",
+        "\n",
+        "# Upload the client library\n",
+        "compressed_file_upload = files.upload()\n",
+        "compressed_file_name = list(compressed_file_upload.keys())[0]\n",
+        "# Decompress the client library\n",
+        "with tarfile.open(compressed_file_name) as tar:\n",
+        "  tar.extractall(path='.')\n",
+        "# Install the client library\n",
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: Remove the lines "\n, from _future_ importabsolute_import\n" through #install the client library\n" included.
we need to use the public packager

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 3]
```diff
@@ -0,0 +1,1204 @@
...
+        "!pip install ./python"
+      ],
+      "execution_count": 1,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 1 with 0
we need to have no previous run

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 4]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close ] on line 177
delete lines 178 through 244

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 5]
```diff
@@ -0,0 +1,1204 @@
...
+        "\n",
+        "# Upload service account key\n",
+        "keyfile_upload = files.upload()\n",
+        "keyfile_name = list(keyfile_upload.keys())[0]\n",
+        "# Authenticate and create an AutoML client.\n",
+        "client = automl_v1beta1.AutoMlClient.from_service_account_file(keyfile_name)\n",
+        "# Authenticate and create a prediction service client.\n",
+        "prediction_client = automl_v1beta1.PredictionServiceClient.from_service_account_file(keyfile_name)"
+      ],
+      "execution_count": 2,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 2 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 6]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ]
remove lines 297 through 324

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 7]
```diff
@@ -0,0 +1,1204 @@
...
+      "cell_type": "code",
+      "source": [
+        "#@title GCP project ID and location\n",
+        "\n",
+        "project_id = 'energy-forecasting' #@param {type:'string'}\n",
+        "location = 'us-central1' #@param {type:'string'}\n",
+        "location_path = client.location_path(project_id, location)\n",
+        "location_path"
+      ],
+      "execution_count": 3,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 3 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 8]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ] and delete lines 367 through 372

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 9]
```diff
@@ -0,0 +1,1204 @@
...
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "'projects/energy-forecasting/locations/us-central1'"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 3
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: repalce 3 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 10]
```diff
@@ -0,0 +1,1204 @@
...
+        "\n",
+        "dataset_display_name = 'energy_forecasting_solution' #@param {type: 'string'}\n",
+        "\n",
+        "create_dataset_response = client.create_dataset(\n",
+        "    location_path,\n",
+        "    {'display_name': dataset_display_name, 'tables_dataset_metadata': {}})\n",
+        "dataset_name = create_dataset_response.name\n",
+        "create_dataset_response"
+      ],
+      "execution_count": 14,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 14 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 11]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ]
delete lines448 though 463

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 12]
```diff
@@ -0,0 +1,1204 @@
...
+              "tables_dataset_metadata {\n",
+              "  stats_update_time {\n",
+              "  }\n",
+              "}"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 14
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 14 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 13]
```diff
@@ -0,0 +1,1204 @@
...
+      "source": [
+        " #@title Import data { vertical-output: true }\n",
+        "\n",
+        "import_data_response = client.import_data(dataset_name, input_config)\n",
+        "print('Dataset import operation: {}'.format(import_data_response.operation))\n",
+        "# Wait until import is done.\n",
+        "import_data_result = import_data_response.result()\n",
+        "import_data_result"
+      ],
+      "execution_count": 16,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 16 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 14]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ] and delete lines 539 through 548

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 15]
```diff
@@ -0,0 +1,1204 @@
...
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              ""
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 16
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 16 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 16]
```diff
@@ -0,0 +1,1204 @@
...
+        "list_table_specs_response = client.list_table_specs(dataset_name)\n",
+        "table_specs = [s for s in list_table_specs_response]\n",
+        "# List column specs\n",
+        "table_spec_name = table_specs[0].name\n",
+        "list_column_specs_response = client.list_column_specs(table_spec_name)\n",
+        "column_specs = {s.display_name: s for s in list_column_specs_response}\n",
+        "[(x, data_types.TypeCode.Name(\n",
+        "  column_specs[x].data_type.type_code)) for x in column_specs.keys()]"
+      ],
+      "execution_count": 17,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 17 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 17]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket and delete lines 613-807

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 18]
```diff
@@ -0,0 +1,1204 @@
...
+              " ('point18_wind_speed', 'FLOAT64'),\n",
+              " ('point18_wind_direction', 'FLOAT64'),\n",
+              " ('point18_pressure', 'FLOAT64'),\n",
+              " ('split', 'CATEGORY')]"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 17
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 17 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 19]
```diff
@@ -0,0 +1,1204 @@
...
+        "update_column_spec_dict = {\n",
+        "    \"name\": column_specs[column_to_category].name,\n",
+        "    \"data_type\": {\n",
+        "        \"type_code\": \"CATEGORY\"\n",
+        "    }\n",
+        "}\n",
+        "update_column_response = client.update_column_spec(update_column_spec_dict)\n",
+        "update_column_response.display_name , update_column_response.data_type \n"
+      ],
+      "execution_count": 18,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 18 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 20]
```diff
@@ -0,0 +1,1204 @@
...
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "('hour', type_code: CATEGORY)"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 18
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 18 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 21]
```diff
@@ -0,0 +1,1204 @@
...
+        "    \"name\": column_specs[column_to_category].name,\n",
+        "    \"data_type\": {\n",
+        "        \"type_code\": \"CATEGORY\"\n",
+        "    }\n",
+        "}\n",
+        "update_column_response = client.update_column_spec(update_column_spec_dict)\n",
+        "update_column_response.display_name , update_column_response.data_type \n"
+      ],
+      "execution_count": 18,
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket [ and delete lines 893 through 898

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 22]
```diff
@@ -0,0 +1,1204 @@
...
+        "    'name': dataset_name,\n",
+        "    'tables_dataset_metadata': {\n",
+        "        'target_column_spec_id': label_column_id,\n",
+        "        'ml_use_column_spec_id': split_column_id,\n",
+        "    }\n",
+        "}\n",
+        "update_dataset_response = client.update_dataset(update_dataset_dict)\n",
+        "update_dataset_response"
+      ],
+      "execution_count": 19,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 19 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 23]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close ] and delete lines 953-958

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 24]
```diff
@@ -0,0 +1,1204 @@
...
+              "    seconds: 1553639685\n",
+              "    nanos: 199000000\n",
+              "  }\n",
+              "}"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 19
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 19 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 25]
```diff
@@ -0,0 +1,1204 @@
...
+        "    \n",
+        "create_model_response = client.create_model(location_path, model_dict)\n",
+        "print('Dataset import operation: {}'.format(create_model_response.operation))\n",
+        "# Wait until model training is done.\n",
+        "create_model_result = create_model_response.result()\n",
+        "model_name = create_model_result.name\n",
+        "create_model_result"
+      ],
+      "execution_count": 0,
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ] and delte lines 1071-1083

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 26]
```diff
@@ -0,0 +1,1204 @@
...
+        "outputId": "42b9296c-d231-4787-f7fb-4aa1a6ff9bd9"
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Model Metrics {vertical-output: true }\n",
+        "\n",
+        "metrics= [x for x in client.list_model_evaluations(model_name)][-1]\n",
+        "metrics.regression_evaluation_metrics"
+      ],
+      "execution_count": 9,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 9 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 27]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ] and delete lines 1104-1112

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 28]
```diff
@@ -0,0 +1,1204 @@
...
+              "root_mean_squared_error: 0.0524103119969368\n",
+              "mean_absolute_error: 0.04162062332034111\n",
+              "mean_absolute_percentage_error: 8.693264961242676\n",
+              "r_squared: 0.5450255274772644"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 9
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 9 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 29]
```diff
@@ -0,0 +1,1204 @@
...
+      "cell_type": "code",
+      "source": [
+        "#@title Feature Importance {vertical-output: true }\n",
+        "\n",
+        "model = client.get_model(model_name)\n",
+        "feat_list = [(x.feature_importance, x.column_display_name) for x in model.tables_model_metadata.tables_model_column_info]\n",
+        "feat_list.sort(reverse=True)\n",
+        "feat_list[:15]"
+      ],
+      "execution_count": 12,
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 12 with 0

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 30]
```diff
@@ -0,0 +1,1204 @@
...
+      "outputs": [
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: close bracket ] and delete lines 1152-1171

File: tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb [hunk 31]
```diff
@@ -0,0 +1,1204 @@
...
+              " (0.009977834299206734, 'point2_pressure'),\n",
+              " (0.009956032037734985, 'point8_wind_gust'),\n",
+              " (0.009465079754590988, 'prev_week_min'),\n",
+              " (0.009312096051871777, 'point7_wind_speed_100m')]"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 12
```
Comment (merla18) [inline tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb]: replace 12 with 0

File: tables/automl/notebooks/retail_product_stockout_prediction/retail_product_stockout_prediction.ipynb [hunk 32]
```diff
@@ -0,0 +1,1041 @@
...
+    {
+      "metadata": {
+        "id": "43aXKjDRt_qZ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Install AutoML Tables client library { vertical-output: true }\n",
+        "\n",
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/retail_product_stockout_prediction.ipynb]: delete lines 162 through 172

File: tables/automl/notebooks/retail_product_stockout_prediction/retail_product_stockout_prediction.ipynb [hunk 33]
```diff
@@ -0,0 +1,1041 @@
...
+        "import tarfile\n",
+        "\n",
+        "# Upload the client library\n",
+        "compressed_file_upload = files.upload()\n",
+        "compressed_file_name = list(compressed_file_upload.keys())[0]\n",
+        "# Decompress the client library\n",
+        "with tarfile.open(compressed_file_name) as tar:\n",
+        "  tar.extractall(path='.')\n",
+        "# Install the client library\n",
+        "!pip install ./python"
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/retail_product_stockout_prediction.ipynb]: replace this line with 
`!pip install google-cloud-automl`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 34]
```diff
@@ -0,0 +1,371 @@
+Copyright 2018 Google LLC 
+Licensed under the Apache License, Version 2.0 (the "License");you may not use this file except in compliance with the License.You may obtain a copy of the License at
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: missing empty line after Copyright, after "at" and after the LICENSE-2.0"

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 35]
```diff
@@ -0,0 +1,371 @@
...
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%201%202019-03-13%20at%201.02.53%20PM.png" width="50%" height="50%">\
+   
+3. Import your dataset or the dataset you downloaded in the last section \
+Click <+New Dataset> → Dataset Name <StockOut> → Click Create Dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%202%202019-03-13%20at%201.05.17%20PM.png" width="50%" height="50%">\
+
+4. You can import data from BigQuery or GCS bucket \
+    a. For BigQuery enter your GCP project ID, Dataset ID and Table ID \
+    After specifying dataset click import dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%203%202019-03-13%20at%201.08.44%20PM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 36]
```diff
@@ -0,0 +1,371 @@
...
+    b. For GCS enter the GCS object location by clicking on BROWSE \
+    After specifying dataset click import dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%204%202019-03-13%20at%201.09.56%20PM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 37]
```diff
@@ -0,0 +1,371 @@
...
+    Depending on the size of the dataset this import can take some time.
+
+5. Once the import is complete you can set the schema of the imported dataset based on your business understanding of the data \
+    a. Select Label i.e. Stockout \
+    b. Select Variable Type for all features \
+    c. Click Continue \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%206%202019-03-13%20at%201.20.57%20PM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 38]
```diff
@@ -0,0 +1,371 @@
...
+
+6. The imported dataset is now analyzed \
+This helps you analyze the size of your dataset, dig into missing values if any, calculate correlation, mean and standard deviation. If this data quality looks good to you then we can move on to the next tab i.e. train. \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%201%202019-03-25%20at%2012.43.13%20AM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 39]
```diff
@@ -0,0 +1,371 @@
...
+
+7. Train \
+    a. Select a model name \
+    b. Select the training budget \
+    c. Select all features you would like to use for training \
+    d. Select optimization objectives. Such as: ROC, Log Loss or PR curve \
+    (As our data is imbalances we use PR curve as our optimization metric) \
+    e. Click TRAIN \
+    f. Training the model can take some time \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%208%202019-03-13%20at%201.34.08%20PM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 40]
```diff
@@ -0,0 +1,371 @@
...
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%202%202019-03-25%20at%2012.44.18%20AM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 41]
```diff
@@ -0,0 +1,371 @@
...
+
+8. Once the model is trained you can click on the evaluate tab \
+This tab gives you stats for model evaluation  \
+         For example our model shows  \
+         Area Under Precision Recall Curve: 0.645  \
+         Area Under ROC Curve: 0.893 \
+         Accuracy: 92.5% \
+         Log Loss: 0.217 \
+Selecting the threshold lets you set a desired precision and recall on your predictions. \
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 42]
```diff
@@ -0,0 +1,371 @@
...
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%203%202019-03-25%20at%2012.49.40%20AM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 43]
```diff
@@ -0,0 +1,371 @@
...
+
+9. Using the model created let's use batch prediction to predict stock-out \
+    a. Batch prediction data inputs could come from BigQuery or your GCS bucket. \
+    b. Select the GCS bucket to store the results of your batch prediction \
+    c. Click Send Batch Predictions \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%2012%202019-03-13%20at%201.56.43%20PM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 44]
```diff
@@ -0,0 +1,371 @@
...
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%2013%202019-03-13%20at%201.59.18%20PM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

File: tables/automl/notebooks/retail_product_stockout_prediction/README.md [hunk 45]
```diff
@@ -0,0 +1,371 @@
...
+
+
+## Building the model using AutoML Tables Python Client Library
+
+In this notebook, you will learn how to build the same model as you have done on the AutoML Tables UI using its Python Client Library.
+
+
+## Evaluation results and business impact
+
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%203%202019-03-25%20at%2012.49.40%20AM.png" width="50%" height="50%">\
```
Comment (merla18) [inline tables/automl/notebooks/retail_product_stockout_prediction/README.md]: remove `< img src="` and replace with `alt text`

**Comment by sirtorry** [comment]
branch has been forked and merged as #2276

**Pull request closed without merging**